clang/lib/DPCT/SaveNewFiles.cpp

```diff
@@ -158,7 +158,7 @@ void rewriteFileName(clang::tooling::UnifiedPath &FileName) {
 
 void rewriteFileName(clang::tooling::UnifiedPath &FileName,
                      const clang::tooling::UnifiedPath &FullPathName) {
-  std::string FilePath;
+  std::string FilePath = FileName.getPath().str();
```

Comment:
```suggestion
  std::string FilePath;
```
As FilePath is used to output by rewriteFileName in Line162.

Reply:
As the new add code shown in Line179, FileName is use to get fileinfo with DpctGlobalInfo::getInstance().insertFile, so it cannot be empty string.